doc/radosgw/pubsub-module.rst

```diff
@@ -35,9 +35,25 @@ be accessed directly, but need to be pulled and acked using the new REST API.
 
    S3 Bucket Notification Compatibility <s3-notification-compatibility>
 
-PubSub Tier Type Configuration
--------------------------------
+PubSub Zone Configuration
+-------------------------
+
+The pubsub sync module requires the creation of a new zone in a `Multisite`_ environment.
+First, a master zone must exist (see: :ref:`master-zone-label`), 
+then a secondary zone should be created (see :ref:`secondary-zone-label`).
+None that, When doing the actual creation of the secondary zone zone, its tier type must be set to ``pubsub``:
```

Comment:
When->when

secondary zone ~~zone~~

Reply:
> aren't these spelled out in PubSub Zone Configuration Parameters section directly above?
i think that this is an alternative to setting the zone parameters in a conf file. these CLI commands are showing how to set the parameters via CLI - will change the headline so it is clear